netjsonconfig/backends/openwrt/converters/wireless.py

```diff
@@ -110,6 +114,16 @@ def __intermediate_encryption(self, wireless):
             uci['key'] = encryption['key']
         # add ciphers
         cipher = encryption.get('cipher')
+        if (
+            protocol == 'wpa3_personal'
+            or protocol == 'wpa2_personal_mixed'
+            or protocol == 'wpa3_enterprise'
+        ):
+            cipher = 'auto'
+        if (protocol == 'wpa3_personal' or protocol == 'wpa3_enterprise') and (
+            'ieee80211w' not in uci or uci['ieee80211w'] == '0'
+        ):
+            uci['ieee80211w'] = '1'
```

Comment:
> I believe this can be simplified as following

Ok, I fixed to use ```if 'wpa3' in protocol:``` to include mixed mode.

> I don't see that it will be wise to overwrite user's configuration (uci['ieee80211w'] == '0'), since we won't be able to reflect this to use in configuration editor.

Ok, removed.